airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -260,6 +284,11 @@ def _update_main_container_yaml(self, container: dict):
             container["image"] = self.image
         if self.image_pull_policy:
             container["imagePullPolicy"] = self.image_pull_policy
+        if self.volume_files_from_secret:
```

Comment:
Maybe this should be a list from a specific type? e.g. custom class that maps the secret config? 
i.e. 
```python
class FilesFromSecret(dict):
  def __init__(self, secret_name:str, **kwargs):
   super().__init__(**kwargs)
     self.secret_name=secret_name
```

And then the input argument is 
```python
volume_files_from_secret: Union[FilesFromSecret, List[FilesFromSecret]] = None
```
Thus allowing someone to send either a single secret or a list of them (initialized as list in __init__ if a single secret load).

Reply:
I thought about it but there are custom classes in the dag when I want to use it feature. And I think it is not good idea.
Add more docs and samples is simpler.

Reply:
OK In that case, please add proper documentation for it so its clear what it does.